CMakeLists.txt

```diff
@@ -138,6 +138,27 @@ option(GOOGLE_CLOUD_CPP_ENABLE_FIRESTORE
 option(GOOGLE_CLOUD_CPP_ENABLE_PUBSUB "Enable building the Pub/Sub library." ON)
 option(GOOGLE_CLOUD_CPP_ENABLE_GENERATOR "Enable building the generator." ON)
 
+set(GOOGLE_CLOUD_CPP_LIBRARIES
+    "bigquery;bigtable;spanner;storage;firestore;pubsub;generator")
+
+# The default list of libraries to build. These can be overridden by the user by
+# passing a comma-separated list, i.e
+# `-DGOOGLE_CLOUD_CPP_ENABLE=spanner,storage`.
+set(GOOGLE_CLOUD_CPP_ENABLE
+    ${GOOGLE_CLOUD_CPP_LIBRARIES}
```

Comment:
My CMake skills are intermediate at best, so I'll ask: Is there a reason to have `GOOGLE_CLOUD_CPP_LIBRARIES` in addition to `GOOGLE_CLOUD_CPP_ENABLE`? Would it work have only the latter variable?

Reply:
I don't remember. Fixed.